### PR TITLE
Add RadioGroup UI variant

### PR DESCRIPTION
**Pull request by mickaelzhang**
## Screenshot / video

Screenshot 2022-09-01 at 16 56 56
Screenshot 2022-09-01 at 16 57 03
Screenshot 2022-09-01 at 16 57 12
Screenshot 2022-09-01 at 16 57 17

## What does this do?

Still in WIP but should be useable now

### CODE REVIEW

File: src/RadioGroup/RadioElement.tsx [hunk 1]
```diff
@@ -22,8 +23,13 @@ export const RadioElement: FC<RadioElementProps> = ({
   value,
   checked,
   onChange,
+  isError,
   ...marginProps
 }) => {
+  console.log({
```
Comment (githab666) [inline src/RadioGroup/RadioElement.tsx:29]: Is there a reason this has been left in?

**Review by githab666 — DISMISSED** [review]
Just one minor comment - looks great! :shipit:

File: src/RadioGroup/RadioElement.tsx (hunk 1 above)
Comment (mickaelzhang) [reply src/RadioGroup/RadioElement.tsx]: Nop, forgot about it, will remove thanks!

**Review by fleonard — APPROVED** [review]
LGTM

**Pull request merged**